pkg/domain/login/auth_test.go

```diff
@@ -64,15 +52,12 @@ func validateToken(t *testing.T, tokenString string, accountID string, tokenSecr
 		return []byte(tokenSecret), nil
 	})
 	if err != nil {
-		t.Errorf("expected no error but got '%v'", err)
+		t.Fatalf("expected no error but got '%v'", err)
 	}
 	claims := token.Claims.(*jwt.RegisteredClaims)
-	if claims.Subject != accountID {
-		t.Errorf("expected '%s' but got '%s'", accountID, claims.Subject)
-	}
-	if claims.ID == "" {
-		t.Error("expected not empty id")
-	}
+	assert.Equalf(t, claims.Subject, accountID, "expected '%s' but got '%s'", accountID, claims.Subject)
+	assert.NotEqual(t, claims.ID, "", "expected not empty id")
```

Comment:
```suggestion
	assert.NotEmpty(t, claims.ID)
```

Depois da uma olhada na biblioteca, tem vários métodos que podem ser usados nos testes. Mas não se prende muito a isso, o mais importante neste momento, acredito que é fazer testes consistentes da API. 😃